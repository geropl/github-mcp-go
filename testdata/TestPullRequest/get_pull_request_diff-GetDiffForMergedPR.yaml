---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 0
        transfer_encoding: []
        trailer: {}
        host: api.github.com
        remote_addr: ""
        request_uri: ""
        body: ""
        form: {}
        headers:
            Accept:
                - application/vnd.github.v3.diff
            User-Agent:
                - go-github/v69.2.0
            X-Github-Api-Version:
                - "2022-11-28"
        url: https://api.github.com/repos/geropl/github-mcp-go-test/pulls/2
        method: GET
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: 222
        uncompressed: false
        body: |
            diff --git a/test-file.md b/test-file.md
            new file mode 100644
            index 0000000..7f59113
            --- /dev/null
            +++ b/test-file.md
            @@ -0,0 +1,3 @@
            +# Test file
            +
            +Created for testing at 2025-03-07T08:33:09Z
            \ No newline at end of file
        headers:
            Access-Control-Allow-Origin:
                - '*'
            Access-Control-Expose-Headers:
                - ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
            Cache-Control:
                - private, max-age=60, s-maxage=60
            Content-Length:
                - "222"
            Content-Security-Policy:
                - default-src 'none'
            Content-Type:
                - application/vnd.github.v3.diff; charset=utf-8
            Date:
                - Fri, 07 Mar 2025 10:06:11 GMT
            Etag:
                - '"ecfc9a5e556f183fb8ff554fbf5948b7547e8ae3a505798dd7bbcfddb4fcc3a0"'
            Github-Authentication-Token-Expiration:
                - 2025-04-06 08:38:40 +0200
            Last-Modified:
                - Fri, 07 Mar 2025 08:33:12 GMT
            Referrer-Policy:
                - origin-when-cross-origin, strict-origin-when-cross-origin
            Server:
                - github.com
            Strict-Transport-Security:
                - max-age=31536000; includeSubdomains; preload
            Vary:
                - Accept, Authorization, Cookie, X-GitHub-OTP,Accept-Encoding, Accept, X-Requested-With
            X-Accepted-Github-Permissions:
                - pull_requests=read; contents=read
            X-Content-Type-Options:
                - nosniff
            X-Frame-Options:
                - deny
            X-Github-Api-Version-Selected:
                - "2022-11-28"
            X-Github-Media-Type:
                - github.v3; param=diff
            X-Github-Request-Id:
                - 8F29:1EA0F8:306A437:320D203:67CAC513
            X-Ratelimit-Limit:
                - "5000"
            X-Ratelimit-Remaining:
                - "3499"
            X-Ratelimit-Reset:
                - "1741342629"
            X-Ratelimit-Resource:
                - core
            X-Ratelimit-Used:
                - "1501"
            X-Xss-Protection:
                - "0"
        status: 200 OK
        code: 200
        duration: 273.085938ms
