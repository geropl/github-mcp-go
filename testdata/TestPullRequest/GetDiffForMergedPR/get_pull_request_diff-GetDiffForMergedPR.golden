{
  "output": "# Pull Request Diff (#2)\n\n```diff\ndiff --git a/test-file.md b/test-file.md\nnew file mode 100644\nindex 0000000..7f59113\n--- /dev/null\n+++ b/test-file.md\n@@ -0,0 +1,3 @@\n+# Test file\n+\n+Created for testing at 2025-03-07T08:33:09Z\n\\ No newline at end of file\n\n```\n",
  "err": ""
}